Frameworks/UIKit/XamlControls.h

```diff
@@ -121,6 +121,12 @@ WXCGrid* CreateLabel();
 WXCTextBlock* GetLabelTextBlock(WXCGrid* labelGrid);
 
 ////////////////////////////////////////////////////////////////////////////////////
+// ScrollViewer
+////////////////////////////////////////////////////////////////////////////////////
+WXCGrid* CreateScrollViewer();
+WXCScrollViewer* GetScrollViewer(WXCGrid* labelGrid);
```

Comment:
good idea. i was a little hesitant here regarding the name. 

---
In reply to: [106712808](https://github.com/Microsoft/WinObjC/pull/2257#discussion_r106712808) [](ancestors = 106712808)